feat: view mode design changes

cooper-joe opened the pull request:
wip

cooper-joe committed:
fix: empty state message design

cooper-joe committed:
fix: title bar design

cooper-joe committed:
fix: background colors, spacing, detail design

cooper-joe committed:
feat: dashboard bar, title bar redesign

martinkrulltott committed:
Merge branch 'master' into design-changes

jenniferarnesen committed:
chore: nightly failing to set up cypress (#2378)

The verify workflow should run on tags, so that a corresponding branch is created
in the d2-ci repository

Do not run the release step on PRs, but do allow it for pushes

yarn cypress install sometimes fails. Turns out it isn't necessary anyway
since the cypress-io gh action takes care of installing cypress.

dhis2-bot committed:
fix(translations): sync translations from transifex (dev)

Automatically merged.

janhenrikoverland committed:
docs: move docs to app (#2384)

dhis2-bot committed:
fix(translations): sync translations from transifex (dev)

Automatically merged.

martinkrulltott committed:
Merge branch 'dev' into design-changes

martinkrulltott commented:
Status update: currently waiting for @janhenrikoverland to review the _design_ changes (not the code itself)

martinkrulltott committed:
refactor: remove unused import

jenniferarnesen committed:
Merge branch 'dev' into design-changes

dependabot[bot] committed:
chore(deps): bump express from 4.18.1 to 4.19.2

Bumps [express](https://github.com/expressjs/express) from 4.18.1 to 4.19.2.
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/master/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.18.1...4.19.2)

---
updated-dependencies:
- dependency-name: express
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

jenniferarnesen committed:
Merge branch 'dev' into dependabot/npm_and_yarn/express-4.19.2

jenniferarnesen committed:
Merge pull request #2944 from dhis2/dependabot/npm_and_yarn/express-4.19.2

chore(deps): bump express from 4.18.1 to 4.19.2

jenniferarnesen committed:
Merge branch 'dev' into design-changes

jenniferarnesen committed:
Merge branch 'master' into design-changes

jenniferarnesen committed:
chore: fix snapshot for ViewDashboard

jenniferarnesen committed:
chore: snapshot fix

jenniferarnesen committed:
chore: another snapshot update

jenniferarnesen committed:
chore: yarn-dedupe

jenniferarnesen committed:
Merge branch 'master' into design-changes

cooper-joe committed:
fix: adjust description design

cooper-joe committed:
fix: move last updated tag into dashboard actions

cooper-joe committed:
fix: title bar changes

cooper-joe commented:
Superseded by https://github.com/dhis2/dashboard-app/pull/3095
